yarn-error.log

```diff
@@ -0,0 +1,6241 @@
+Arguments: 
```

Comment:
This file contains error logs, so it should not be added to the git projet.
Would you please remove it ?